experiments/evals/evals.py

```diff
@@ -321,6 +333,63 @@ def default_base_eval(
     return eval_jobs
 
 
+def default_sft_eval(
+    step: ExecutorStep | InputName | str,
+    resource_config: ResourceConfig = SINGLE_TPU_V6E_8,
+    max_eval_instances: int | None = None,
+    engine_kwargs: dict | None = DEFAULT_LM_EVAL_MODEL_KWARGS,
+    run_generation_evals: bool = True,
+    apply_chat_template: bool = True,
+):
+    # Set up evaluations for core tasks (including GPQA)
+    eval_jobs = []
+    leaderboard_grouped = default_eval(
+        step=step,
+        resource_config=resource_config,
+        evals=OPEN_LM_LEADERBOARD_MCQ,
+        apply_chat_template=apply_chat_template,
+    )
+    eval_jobs.append(leaderboard_grouped)
+
+    # Run tasks where we report Macro_Avg separately to make sure the macro avg gets computed correctly.
+
+    mmlu_5shot = default_eval(
+        step=step, resource_config=resource_config, evals=(MMLU_5_SHOT,), apply_chat_template=apply_chat_template
+    )
+    eval_jobs.append(mmlu_5shot)
+
+    mmlu_pro_5shot = default_eval(
+        step=step, resource_config=resource_config, evals=(MMLU_PRO_5_SHOT,), apply_chat_template=apply_chat_template
+    )
+    eval_jobs.append(mmlu_pro_5shot)
+
+    name, model_step_path = extract_model_name_and_path(step)
+    if run_generation_evals:
+        leaderboard_generation = evaluate_lm_evaluation_harness(
+            name,
+            model_step_path,
+            KEY_GENERATION_TASKS,
+            max_eval_instances=max_eval_instances,
+            engine_kwargs=engine_kwargs,
+            resource_config=resource_config,
+            apply_chat_template=apply_chat_template,
+        )
+
+        eval_jobs.append(leaderboard_generation)
+
+        olmo_generation = evaluate_lm_evaluation_harness(
```

Comment:
why is this called olmo_generation?

Reply:
I guess you mean openlm_generation

Reply:
These are called OLMo generation because we added them for eval parity with what OLMo 2 reports in their blog/paper even though they aren't in the HuggingFace LM Leaderboard